### PR TITLE
Rubocop

### PR DESCRIPTION
**Pull request by AgaDufrat**
4 offences remaining, which had to be kept in:
```
spec/models/user_spec.rb:3:1: C: Metrics/BlockLength: Block has too many lines. [57/25]
describe User do ...
^^^^^^^^^^^^^^^^
spec/models/user_spec.rb:4:24: C: Style/BlockDelimiters: Avoid using {...} for multi-line blocks.
  let(:user_db_params) {
                       ^
spec/models/user_spec.rb:8:25: C: Style/BlockDelimiters: Avoid using {...} for multi-line blocks.
  let(:user_obj_params) {
                        ^
spec/models/user_spec.rb:63:3: C: Metrics/AbcSize: Assignment Branch Condition size for check_user is too high. [16/15]
  def check_user(user) ...
  ^^^^^^^^^^^^^^^^^^^^
```

### CODE REVIEW

File: lib/models/user.rb [hunk 1]
```diff
@@ -3,38 +3,43 @@
...
     result = select_by(column: 'email', value: email)
-    if result && password_check(password: password, result: result)
-        build_user(result)
-    else
-      nil
-    end
+    build_user(result) if result && password_check(
+      password: password,
+      result: result
+    )
```
Comment (willmcb) [inline lib/models/user.rb:31]: Does result return nil in this case as other code depends on it returning `nil`?

File: spec/models/user_spec.rb [hunk 2]
```diff
@@ -1,52 +1,58 @@
...
 describe User do
-  let(:user_db_params) { {name: "Name1", username: "Username", email: "email@email.com", password: "password"} }
-  let(:user_obj_params) { {id: "1", name: "Name1", username: "Username", email: "email@email.com"} }
+  let(:user_db_params) {
+    { name: 'Name1', username: 'Username',
+      email: 'email@email.com', password: 'password' }
+  }
+  let(:user_obj_params) {
+    { id: '1', name: 'Name1', username: 'Username', email: 'email@email.com' }
+  }
```
Comment (willmcb) [inline spec/models/user_spec.rb]: this should be 
```ruby
 let(:user_db_params)  do
    { name: 'Name1', username: 'Username',
      email: 'email@email.com', password: 'password' }
  end

 let(:user_obj_params) do
    { id: '1', name: 'Name1', username: 'Username', email: 'email@email.com' }
  end
```

**Review by willmcb — CHANGES_REQUESTED** [review]
Please see comments

**Comment by AgaDufrat** [comment]
below offence is fixed in spaces branch
```
spec/models/user_spec.rb:63:3: C: Metrics/AbcSize: Assignment Branch Condition size for check_user is too high. [16/15]
  def check_user(user) ...
  ^^^^^^^^^^^^^^^^^^^^
```

**Review by willmcb — APPROVED** [review]

**Comment by willmcb** [comment]
:shipit:

**Pull request merged**